test/testdata/fixtures/reaper-control-plane/reaper.yaml

```diff
@@ -17,3 +17,5 @@ spec:
     enabled: false
   uiUserSecretRef:
     name: reaper-ui-secret
+  containerImage:
```

Comment:
Question: Do we still need this or have we now done a release? I suggest you put a TODO comment here so that it gets (hopefully) picked up before release.

Reply:
We only merged REAP-10 yesterday, we do not have a Reaper release with this just yet.